pkg/scalers/datadog_scaler.go

```diff
@@ -174,8 +197,12 @@ func (s *datadogScaler) Close(context.Context) error {
 	return nil
 }
 
-// IsActive returns true if we are able to get metrics from Datadog
 func (s *datadogScaler) IsActive(ctx context.Context) (bool, error) {
+	return true, nil
```

Comment:
Maybe I'm wrong (correct me in that case @zroubalik ), but if the scaler returns `true` always in this function, this scaler can't scale to zero. 
Is this intended? We should add a note in docs in that case

Reply:
Yeah, it shouldn't behave like that. THe `IsActive` should mark the activity of the scaler - ie whether the target vaules in Datadog are above (Active) or below (Not Active) the threshold.